index.js

```diff
@@ -1,32 +1,25 @@
+/* jshint node: true */
 'use strict';
 
-var path = require('path');
-var fs = require('fs');
 var autoprefixer = require('broccoli-autoprefixer');
-var defaults = require('lodash').defaults;
+var defaults     = require('lodash').defaults;
 
-function EmberCLIAutoprefixer(project) {
-  this.project = project;
-  this.name = 'Ember CLI Autoprefixer';
-}
+module.exports = {
+  name: 'ember-cli-autoprefixer',
+  included: function(app, parentAddon) {
+    this.app = app;
+    this.options = defaults(this.app.options.autoprefixer || {}, {
+      enabled: true
+    });
+    this.enabled = this.options.enabled;
+  },
+  postprocessTree: function(type, tree) {
+    var tree;
```

Comment:
@kimroen Aghh good catch there, It was working so I presume it wasn't overriding but I should remove that var declaration